browser/resources/settings/email_aliases_page/content/email_aliases_modal.tsx

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import Button from "@brave/leo/react/button"
+import Icon from "@brave/leo/react/icon"
+import { getLocale } from '$web-common/locale'
+import formatMessage from '$web-common/formatMessage'
+import * as React from 'react'
+import { ViewState, ViewMode, MappingService, MAX_ALIASES } from "./types"
+import Col from "./styles/Col"
+import Row from "./styles/Row"
+import Input from "@brave/leo/react/input"
+import styled from "styled-components"
+import onEnterKey from "./onEnterKey"
+import { color, font, spacing } from "@brave/leo/tokens/css/variables"
+
+
+const ModalSectionCol = styled(Col)`
+  margin: ${spacing["2Xl"]} 0;
+  & h3 {
+    margin: ${spacing.s} ${spacing.m};
+  }
+  & leo-input {
+    margin: ${spacing.s} 0;
+  }
+`
+
+const ButtonRow = styled(Row)<{ bubble: boolean }>`
+  justify-content: ${props => props.bubble ? 'space-between' : 'end'};
+  margin: ${spacing["2Xl"]} 0;
+  & leo-button {
+    flex-grow: 0;
+  }
+  & span {
+    display: flex;
+    flex-direction: row;
+    column-gap: ${spacing.m};
+  }
+`
+
+const GeneratedEmailContainer = styled(Row)`
+  font: ${font.large.regular};
+  background-color: ${color.neutralVariant[10]};
+  border-radius: ${spacing.m};
+  padding: 0 0 0 ${spacing.m};
+  margin: ${spacing.s} 0;
+  justify-content: space-between;
+  height: ${spacing["5Xl"]};
```

Comment:
ideally, don't set an explicit height - it should come from the content

Reply:
The figma design seems to have set an explicit height of 44px. So I updated it to that.